src/cljam/io/util.clj

```diff
@@ -136,38 +140,3 @@
     #"(?i)\.bcf$" :bcf
     #"(?i)\.bed" :bed
     (throw (IllegalArgumentException. "Invalid file type"))))
-
-(defn ^Closeable reader
```

Comment:
That causes cyclic load. `cljam.sam` loads `cljam.io.util` for `cljam.sam/reader`, and `cljam.io.util` loads `cljam.sam` for `cljam.io.util/reader`. If `reader/writer` are realy necessary, I have to reconsider namespace allocation.